sdk/communication/Azure.Communication.Rooms/README.md

```diff
@@ -52,8 +47,96 @@ RoomsClient client = new RoomsClient(connectionString);
+List<RoomParticipant> invitedParticipants = new List<RoomParticipant>
+{
+    participant1,
+    participant2
+};
+
+Response<CommunicationRoom> createRoomResponse = await roomsClient.CreateRoomAsync(validFrom, validUntil, invitedParticipants);
+CommunicationRoom createCommunicationRoom = createRoomResponse.Value;
+```
+
+### Update a room
+The `validFrom` and `validUntil` properties of a created room can be updated by calling the `UpdateRoom` or `UpdateRoomAsync` function from `RoomsClient`.
+
+```C# Snippet:Azure_Communication_Rooms_Tests_Samples_UpdateRoomAsync
+validUntil = validFrom.AddDays(30);
+Response<CommunicationRoom> updateRoomResponse = await roomsClient.UpdateRoomAsync(createdRoomId, validFrom, validUntil);
+CommunicationRoom updateCommunicationRoom = updateRoomResponse.Value;
+```
+
+### Get a created room
+A created room can be retrieved by calling the `GetRoom` or `GetRoomAsync` function from `RoomsClient` and passing in the associated `roomId`.
+
+```C# Snippet:Azure_Communication_Rooms_Tests_Samples_GetRoomAsync
+Response<CommunicationRoom> getRoomResponse = await roomsClient.GetRoomAsync(createdRoomId);
+CommunicationRoom getCommunicationRoom = getRoomResponse.Value;
+```
+
+### Delete room
+To delete a room, call the `DeleteRoom` or `DeleteRoomAsync` function from RoomsClient.
+```C# Snippet:Azure_Communication_Rooms_Tests_Samples_DeleteRoomAsync
+Response deleteRoomResponse = await roomsClient.DeleteRoomAsync(createdRoomId);
+```
+
+### Add Or update participants in a room
+In order to add new participants or update existing participants, call the `AddOrUpdateParticipants` or `AddOrUpdateParticipantsAsync` function from RoomsClient.
+
+```C# Snippet:Azure_Communication_Rooms_Tests_Samples_AddOrUpdateParticipants
+CommunicationIdentifier communicationUser3 = communicationIdentityClient.CreateUserAsync().Result.Value;
+RoomParticipant newParticipant = new RoomParticipant(communicationUser3) { Role = ParticipantRole.Consumer };
+
+// Previous snippet for create room added participant2 as Presenter
+participant2 = new RoomParticipant(communicationUser2) { Role = ParticipantRole.Attendee };
 
-The following sections provide several code snippets covering some of the most common tasks, which is available at Sample1_RoomsRequests.md
+List<RoomParticipant> participantsToAddOrUpdate = new List<RoomParticipant>
+{
+    participant2,   // participant2 updated from Presenter to Attendee
+    newParticipant, // newParticipant added to the room
+};
+
+Response addOrUpdateParticipantResponse = await roomsClient.AddOrUpdateParticipantsAsync(createdRoomId, participantsToAddOrUpdate);
+```
+
+### Remove participants in a room
+To remove participants from a room, call the `RemoveParticipants` or `RemoveParticipantsAsync` function from RoomsClient.
+
+```C# Snippet:Azure_Communication_Rooms_Tests_Samples_RemoveParticipants
+List<CommunicationIdentifier> participantsToRemove = new List<CommunicationIdentifier>
```

Comment:
Please use IEnumerable

Reply:
Discussed offline, we will keep as list as Contoso may want to modify their list after initial intialization